custom_components/storj/__init__.py

```diff
@@ -22,23 +22,27 @@
 async def async_setup_entry(hass: HomeAssistant, entry: StorjConfigEntry) -> bool:
```

Comment:
If everything went right in the config flow, I'm not sure if there's anything that could go wrong during/after setup that we could check for here.